test/e2e/common/pod_resize.go

```diff
@@ -349,18 +394,266 @@ var _ = ginkgo.Describe("[sig-node] PodInPlaceResize", func() {
+			containers: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "100m", CPULim: "400m", MemReq: "128Mi", MemLim: "512Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
+				},
+			},
+			patchString: `{"spec":{"containers":[
+				{"name":"c1", "resources":{"requests":{"cpu":"200m","memory":"256Mi"}}}
+			]}}`,
+			expected: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "200m", CPULim: "400m", MemReq: "256Mi", MemLim: "512Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
+				},
+			},
+		},
+		{
+			name: "Bustable class pod, one container - decrease CPU & memory requests",
+			containers: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "200m", CPULim: "400m", MemReq: "256Mi", MemLim: "512Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
+				},
+			},
+			patchString: `{"spec":{"containers":[
+				{"name":"c1", "resources":{"requests":{"cpu":"100m","memory":"128Mi"}}}
+			]}}`,
+			expected: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "100m", CPULim: "400m", MemReq: "128Mi", MemLim: "512Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
+				},
+			},
+		},
+		// By chenw, adding E2E test cases 6 for Burstable class single container Pod that specifies both CPU & memory
+		{
+			name: "Bustable class pod, one container - increase CPU & memory limits",
+			containers: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "100m", CPULim: "200m", MemReq: "128Mi", MemLim: "256Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
+				},
+			},
+			patchString: `{"spec":{"containers":[
+				{"name":"c1", "resources":{"limits":{"cpu":"400m","memory":"512Mi"}}}
+			]}}`,
+			expected: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "100m", CPULim: "400m", MemReq: "128Mi", MemLim: "512Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
+				},
+			},
+		},
+		{
+			name: "Bustable class pod, one container - decrease CPU & memory limits",
+			containers: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "100m", CPULim: "400m", MemReq: "128Mi", MemLim: "512Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
+				},
+			},
+			patchString: `{"spec":{"containers":[
+				{"name":"c1", "resources":{"limits":{"cpu":"200m","memory":"256Mi"}}}
+			]}}`,
+			expected: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{CPUReq: "100m", CPULim: "200m", MemReq: "128Mi", MemLim: "256Mi"},
+					CPUPolicy: &noRestart,
+					MemPolicy: &noRestart,
+					RestartCnt: 0,
```

Comment:
we can get rid of CPUPolicy, MemPolicy from the table and set them below in the test for loop inside ginkgo.It (or ginkgo.JustBeforeEach() if that works ). Something like:
```if tc.policy == nil tc.policy = &noRestart```

ExpectedRestartCount should default to 0 if not specified.